content/articles/aes-rsa-encryption/index.md

```diff
@@ -0,0 +1,138 @@
+Encryption is a way of enciphering data especially when it's on transit to prevent unauthorized personnel from gaining access to it. Encryption is always done with the intent of allowing only the intended recipient of a piece of data to have access to the data. Maintaining the integrity, confidentiality, and availability of data and information is key in data communication and this is where data encryption comes in handy.
+
+### Table of contents
+1. [Prerequisites](#prerequisites)
+2. [What is an encryption algorithm?](#what-is-an-encryption-algorithm)
+3. [Getting started with AES encryption algorithm](#getting-started-with-aes-encryption-algorithm)
+4. [Functionality of AES algorithm](#functionality-of-aes-algorithm)
+5. [Getting started with RSA encryption algorithm](#getting-started-with-rsa-encryption-algorithm)
+6. [RSA algorithm functionality](#rsa-algorithm-functionality)
+7. [Conclusion](#conclusion)
+
+### Prerequisites
+
+For a reader to understand this article, they should have the following prior knowledge:
+
+- A basic understanding of the concept of data encryption and the need for encryption in data communication.
+- A beginner's understanding of data cryptography.
```

Comment:
You've written an article on data cryptography before, right? Please link it where you need the reader to have an understanding of data cryptography